lib/debezium/types.go

```diff
@@ -58,28 +56,6 @@ const (
 	KafkaDecimalPrecisionKey = "connect.decimal.precision"
 )
 
-// toBytes attempts to convert a value (type []byte, or string) to a slice of bytes.
-// - If value is already a slice of bytes it will be directly returned.
-// - If value is a string we will attempt to base64 decode it.
-func toBytes(value any) ([]byte, error) {
```

Comment:
Moved

Reply:
I think https://github.com/artie-labs/transfer/pull/827/files#diff-6f8a144d4abeb92f8240e55383e15b8323e83bb3b9f69088274ed61f4f344e96R117 can probably go away too.

Reply:
The reason `toBytes` is here is because I was going to use it in this [switch statement](https://github.com/artie-labs/transfer/pull/827/files#diff-6f8a144d4abeb92f8240e55383e15b8323e83bb3b9f69088274ed61f4f344e96R85) so that if `Field.Type` is  `Bytes` we do the base64 decoding once at the field level and then just pass bytes to the individual value converter, similar to how we do float to integer conversion for the integer types. 

The goal was to be able to get rid of [this comment](https://github.com/artie-labs/transfer/pull/827/files#diff-6f8a144d4abeb92f8240e55383e15b8323e83bb3b9f69088274ed61f4f344e96R119) once we'd verified that bytes worked everywhere downstream.

Reply:
Posting here for posterity as I'm not going to get to it right away.

The idea here is that we should handle the `toBytes` in the pre-processing layer, when we detect the incoming payload is `bytes`.

We can do that as a first pass, then the Debezium converters can expect bytes